src/prefect/cli/deployment.py

```diff
@@ -920,6 +920,13 @@ async def build(
             f"Deployment '{deployment.flow_name}/{deployment.name}' successfully created with id '{deployment_id}'.",
             style="green",
         )
+        connection_status = await check_orion_connection()
+        ui = ui_base_url(connection_status)
+
+        if ui:
+            app.console.print(
+                f"View Deployment in UI: {ui}/deployments/deployment/{deployment_id}"
+            )
```

Comment:
We're doing this get connection, get ui base url, create full url pattern in a quite a few places now, but be worth a `build_ui_url` method in `orion_utils`.

Reply:
Does `build_ui_url` include both `check_orion_connection` and `ui_base_url`?

Reply:
That's what I was thinking, but feel free to ignore :)

Reply:
This is being done in #7411